mlflow/store/artifact/azure_blob_artifact_repo.py

```diff
@@ -49,31 +49,32 @@ def __init__(self, artifact_uri, client=None):
                     "Please install it via: pip install azure-identity"
                 ) from exc
 
-            account_url = "https://{account}.blob.core.windows.net".format(account=account)
+            account_url = "https://{account}.{api_uri_suffix}".format(account=account, api_uri_suffix=api_uri_suffix)
             self.client = BlobServiceClient(
                 account_url=account_url, credential=DefaultAzureCredential()
             )
 
     @staticmethod
     def parse_wasbs_uri(uri):
-        """Parse a wasbs:// URI, returning (container, storage_account, path)."""
+        """Parse a wasbs:// URI, returning (container, storage_account, path, api_uri_suffix)."""
         parsed = urllib.parse.urlparse(uri)
         if parsed.scheme != "wasbs":
             raise Exception("Not a WASBS URI: %s" % uri)
-        match = re.match("([^@]+)@([^.]+)\\.blob\\.core\\.windows\\.net", parsed.netloc)
+        match = re.match(r"([^@]+)@([^.]+)\.(blob\.core\.(windows\.net|chinacloudapi\.cn))", parsed.netloc)
         if match is None:
             raise Exception(
-                "WASBS URI must be of the form " "<container>@<account>.blob.core.windows.net"
+                "WASBS URI must be of the form " "<container>@<account>.blob.core.windows.net" " or <container>@<account>.blob.core.chinacloudapi.cn"
```

Comment:
```suggestion
                "WASBS URI must be of the form <container>@<account>.blob.core.windows.net"
                " or <container>@<account>.blob.core.chinacloudapi.cn"
```

Can we fix this line?

Reply:
fixed, not sure why pylint not work locally.